webpack.config.js

```diff
@@ -178,13 +178,16 @@ function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv )
 				},
 				{
 					test: /\.(sc|sa|c)ss$/,
-					use: _.compact( [
+					use: [
 						MiniCssExtractPlugin.loader,
 						'css-loader',
 						{
 							loader: 'postcss-loader',
 							options: {
-								plugins: [ require( 'autoprefixer' ) ],
+								plugins: _.compact( [
+									require( 'autoprefixer' ),
+									! isDevelopment && require( 'cssnano' ),
```

Comment:
Great 👍

Just a small question, does it make sense to put all postcss configuration into a `.postcssrc.js`? We seem to have a `.postcssrc.json` already - not sure though if that fits well with our current approach or if such a file is even recognized by the `postcss-loader`.

Feel free to ignore this comment if it gets in the way.

Reply:
That came previously up in https://github.com/Automattic/wp-calypso/pull/26724#discussion_r210834869

I'd prefer having build configs as much at one place as possible to make understanding them easier, especially when:
- it's just two lines
- it would contain just part of the CSS related configs

Reply:
I see your point @simison and I agree. If I understand you correctly we'd remove our `.postcssrc.json` after we migrated everything to webpack?

Reply:
> we'd remove our `.postcssrc.json` after we migrated everything to webpack?

Correct 👍 

`.postcssrc.json` is currently needed only by `npm` scripts.

I think `postcss-loader` in Webpack _does_ pick that config file up, but since it contains only `plugins` and nothing else, it gets overwritten with `options.plugins` from `postcss-loader`.